spdx/licenses.go

```diff
@@ -62,6 +62,7 @@ var osi = []string{
 	"AGPL-3.0",
 	"GPL-2.0",
 	"GPL-3.0",
+	"GPL-3.0-or-later",
```

Comment:
We should probably add a bunch of other identifiers that added in the latest SPDX license list, but I can take care of this once we merge your change.